docs/tutorials/development-setup.md

```diff
@@ -31,8 +30,13 @@ pip install uv
 git clone https://github.com/mbailey/voicemode
 cd voicemode
 
-# Install in development mode
-uv tool install -e .
```

Comment:
Why do you prefer this over `uv tool install -e  .`?

Reply:
actually,  Yes, if you want to have a virtual environment we need the uv venv and activate line.   Because this is development mode, we should keep use the uv pip install -e .[dev,test] and drop the uv pip install -e .